pydantic/functional_validators.py

```diff
@@ -26,7 +26,7 @@
 
 @dataclasses.dataclass(frozen=True, **_internal_dataclass.slots_true)
 class AfterValidator:
-    """Usage docs: https://docs.pydantic.dev/2.10/concepts/validators/#annotated-validators
+    """Usage docs: https://docs.pydantic.dev/2.10/concepts/validators/#field-validators
```

Comment:
We have a [plugin](https://github.com/pydantic/pydantic/blob/main/docs/plugins/griffe_doclinks.py) that enforces linking between these sections, and it needs the anchor to be a heading.

Reply:
Hmph. Unfortunate. How much effort would be required to support this?

Reply:
Well imo we should probably drop this plugin. It adds an admonition below the related heading to link API documentation, but we could do it manually. Sometimes it's also not necessary because we do something like:

``````markdown
## The validate_call decorator

??? API docs
    [`validate_call()`][pydantic.validate_call]

The [`validate_call()`][pydantic.validate_call] decorator ...
// The link to validate_call is repeated here
``````

And not having the plugin manually adding the crosslink could give more flexibility, like in this case where we don't have a proper heading to attach to.

Reply:
Fair enough, we can leave as is then and drop the plugin in a separate PR?